pex.go

```diff
@@ -144,8 +145,11 @@ func (me *pexMsgFactory) PexMsg() *pp.PexMsg {
 // Per-torrent PEX state
 type pexState struct {
 	sync.RWMutex
-	tail *pexEvent     // event feed list
-	hold []pexEvent    // delayed drops
+	tail *pexEvent  // event feed list
+	hold []pexEvent // delayed drops
```

Comment:
This one's also unrelated to the throttling: it's needed to maintain the minimum of 25 always visible peers when the current connection count is below 25, also as provisioned in the BEP. Also good to recover the logic if it got lost. There also should be code elsewhere which uses this field and which are not being added in this PR.